test/arrays/diten/arrayAsVecShrinkBug.good

```diff
@@ -1,21 +1,21 @@
-push_back reallocate: 1..0 => 1..1 (1..1)
-push_back reallocate: 1..1 => 1..2 (1..2)
-push_back reallocate: 1..2 => 1..3 (1..3)
-push_back reallocate: 1..3 => 1..4 (1..4)
-push_back reallocate: 1..4 => 1..6 (1..5)
-push_back reallocate: 1..6 => 1..9 (1..7)
-push_back reallocate: 1..9 => 1..13 (1..10)
-push_back reallocate: 1..13 => 1..19 (1..14)
-push_back reallocate: 1..19 => 1..28 (1..20)
-push_back reallocate: 1..28 => 1..42 (1..29)
-push_back reallocate: 1..42 => 1..63 (1..43)
-push_back reallocate: 1..63 => 1..94 (1..64)
-push_back reallocate: 1..94 => 1..141 (1..95)
-push_back reallocate: 1..141 => 1..211 (1..142)
-push_back reallocate: 1..211 => 1..316 (1..212)
-push_back reallocate: 1..316 => 1..474 (1..317)
-push_back reallocate: 1..474 => 1..711 (1..475)
-push_back reallocate: 1..711 => 1..1066 (1..712)
+reallocateArray: 1..0 => 1..1 (1..1)
```

Comment:
I think it would be good to preserve which method called reallocateArray.

Reply:
Actually, doing that would require undoing the refactor, since the check for whether or not an allocation occurs within `reallocateArray`.